Update of abort blocks documentation (#513)
0xSpaceShard/starknet-devnet-rs: website/docs/blocks.md

## Abort blocks

> This functionality allows simulating block abortion that can occur on mainnet, requires `-state-archive-capacity full` CLI option.

Reviewer: A dash is missing, but the clarity could be improved, too:
```suggestion
This functionality allows simulating block abortion that can occur on mainnet. It is supported in the `--state-archive-capacity full` mode.
```

Author: 34a300fdfc68193e7e92117ce93e7dde11f97e45

You can abort blocks and revert transactions from the specified block to the currently latest block. Newly created blocks after the abortion will have accepted status and will continue with numbering where the last accepted block left off.

The state of Devnet will be reverted to the state of the last accepted block.

E.g. assume there are 3 accepted blocks numbered 1, 2 and 3. Upon receiving a request to abort blocks starting with block 2, the blocks numbered 2 and 3 are aborted and their transactions reverted. The state of network will be as it was in block 1. Once a new block is mined, it will be accepted and it will have number 2.

> Aborted blocks can only be queried by block hash. Aborting the blocks in forking origin and already aborted blocks is not supported and results in an error. Aborting of devnet's genesis block is not supported. 

Author: 34a300fdfc68193e7e92117ce93e7dde11f97e45
